_commands/basics/wc.md

```diff
@@ -0,0 +1,84 @@
+---
+---
+
+wc
+-------
+
+Print the number of bytes, words, and lines in files
+
+~~~ bash
+a.txt: 
```

Comment:
Would like to see this (and similar others in your doc) more like Pat formats it in homework... e.g.

    $ cat a.txt
    hello
    $ wc a.txt
    1